assets/js/blocks/product-collection/edit.tsx

```diff
@@ -10,11 +10,12 @@ import { useEffect } from '@wordpress/element';
  * Internal dependencies
  */
 import { ImageSizing } from '../../atomic/blocks/product-elements/image/types';
-import { ProductCollectionAttributes } from './types';
+import { ProductCollectionAttributes, ProductCollectionQuery } from './types';
```

Comment:
Nit: should we intentionally use `import type` here?
```suggestion
import type { ProductCollectionAttributes, ProductCollectionQuery } from './types';
```

Reply:
Although I don't have a strong opinion about this so I will go ahead and add the `type` with import. It might help in clarifying the intent. 🤝